api/application.go

```diff
@@ -1061,8 +1067,13 @@ func (h ApplicationHandler) AssessmentCreate(ctx *gin.Context) {
 	if newAssessment {
 		metrics.AssessmentsInitiated.Inc()
 	}
+	questionnaires, err := assessment.NewQuestionnaireResolver(h.DB(ctx))
```

Comment:
The questionnaire is already loaded from the database on lines 1038-1040, so you can grab the property from that instead of setting up a resolver.